pkg/apiserver/handlers/loglevel/handler.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package loglevel
+
+import (
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/log"
+)
+
+// HandleFunc returns the function which can handle the /log-level API request.
```

Comment:
You used `/loglevel` for the APIU endpoint

Reply:
I like to follow the kubectl (and other antctl commands) style which use "-" to connect two words in command names, like cluster-info, port-forward, api-resources.

Reply:
I undertand and I'm not suggesting you changed the antctl command name. But you are referring to the API endpoint here, for which you used `/loglevel` not `/log-level`.

Reply:
I see. I think we want not connectors in API path?

Reply:
We haven't been using them so far, so I was really just suggesting fixing the comment

Reply:
Ok. Sorry for being misunderstanding your comments.
Changed to /loglevel.